README.md

```diff
@@ -9,31 +9,41 @@ JWST Calibration Pipeline
 
 Note
 ----
-Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
+Beginning with version 0.9.0, **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest stable version of the library, we recommend using [conda](https://conda.io/docs/index.html)
+and the [astroconda](https://astroconda.readthedocs.io) channel to install the
+software and using an conda environemnt for the installation:
 
-    % cd jwst
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda create -n jwst python=3
```

Comment:
The only "sane" way to install development packages without breaking your entire conda ecosystem is:
```
$ conda create -n jwst --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults jwst
```

_NOTE:_ Installing `jwst` from the development channel like this will always yield the latest `master` build. In order to install a `stable` package, you need to supply the exact version: `jwst=9.0.0.dev0`.

_NOTE_: Because the package, channel, and dependencies are marching forward, this might not work as intended anyway. You *could* point people to the "stable" development snapshots under https://github.com/astroconda/astroconda-releases instead.